components/skus/browser/rs/lib/src/sdk/orders.rs

```diff
@@ -206,8 +222,47 @@ where
         Ok(order)
     }
 
+    #[instrument]
+    // submit_receipt allows for order proof of payment
+    pub async fn submit_receipt(
+        &self,
+        order_id: &str,
+        receipt: &str,
+    ) -> Result<SubmitReceipt, SkusError> {
```

Comment:
Since the data in `SubmitReceipt` is never used, let's change the return type here to `Result<(), SkusError>`

Reply:
https://github.com/brave/brave-core/pull/13820/commits/9ca2f25358569a798b005005172d789a722eabdc